src/lib/apiTypesV2/SubscriptionExpression.h

```diff
@@ -40,8 +43,9 @@ struct SubscriptionExpression
   std::string               geometry;
   std::string               coords;
   std::string               georel;
+
+  StringFilter              stringFilter;
```

Comment:
The `std::string q` field reamins in this class. I guess that it is due that the "string nature of q" is needed for some operations (e.g. persiting in DB). Maybe a comment about this could be useful (at a first glance, it may be a bit confusing have stringFilter in addition to q).

Or maybe I'm being too paranoid with documentation stuff :). If you find it self-evident, ignore this comment with a NTC.